canonical/ubuntu.com
# /landscape copy updates

## Done

- Updated copy as per docs linked below
- Ran djlint on /landscape/install

## QA

- View the site locally in your web browser at: https://ubuntu-com-13830.demos.haus/landscape and check against [doc](https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit)
- View  https://ubuntu-com-13830.demos.haus/landscape/install and check against [doc](https://docs.google.com/document/d/1tU1z9PUm_kSwGJEsopJibE4gI8Gfv_3Dw_qN2_io_ao/edit#heading=h.nscizu2nycw8)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10794
